pkg/da/data_availability_header_test.go

```diff
@@ -47,31 +47,32 @@ func TestNewDataAvailabilityHeader(t *testing.T) {
 		{
 			name: "typical",
 			expectedHash: []byte{
-				0xfe, 0x9c, 0x6b, 0xd8, 0xe5, 0x7c, 0xd1, 0x5d, 0x1f, 0xd6, 0x55, 0x7e, 0x87, 0x7d, 0xd9, 0x7d,
-				0xdb, 0xf2, 0x66, 0xfa, 0x60, 0x24, 0x2d, 0xb3, 0xa0, 0x9c, 0x4f, 0x4e, 0x5b, 0x2a, 0x2c, 0x2a,
+				0x57, 0x71, 0xc6, 0x77, 0x2f, 0x32, 0x95, 0x73, 0xaa, 0xb8, 0x20, 0xd1, 0xbe, 0x4c, 0xc2, 0x21,
+				0x7d, 0x54, 0xb6, 0x7e, 0xf2, 0x4f, 0xbc, 0xd3, 0x9a, 0x95, 0x15, 0xd0, 0x92, 0x63, 0xc1, 0xf9,
 			},
 			squareSize: 2,
 			shares:     generateShares(4, 1),
 		},
 		{
 			name: "max square size",
 			expectedHash: []byte{
-				0xe2, 0x87, 0x23, 0xd0, 0x2d, 0x54, 0x25, 0x5f, 0x79, 0x43, 0x8e, 0xfb, 0xb7, 0xe8, 0xfa, 0xf5,
-				0xbf, 0x93, 0x50, 0xb3, 0x64, 0xd0, 0x4f, 0xa7, 0x7b, 0xb1, 0x83, 0x3b, 0x8, 0xba, 0xd3, 0xa4,
+				0xbf, 0xe5, 0x8f, 0x4b, 0xae, 0x2b, 0x65, 0x8b, 0xa8, 0xcb, 0xf9, 0xee, 0x8c, 0x6a, 0x1f, 0x72,
+				0xa9, 0x58, 0xc4, 0xcc, 0xca, 0x41, 0x4c, 0xbf, 0x8b, 0x18, 0xf9, 0x53, 0xe, 0xb1, 0x40, 0x54,
 			},
 			squareSize: appconsts.MaxSquareSize,
 			shares:     generateShares(appconsts.MaxSquareSize*appconsts.MaxSquareSize, 99),
 		},
 	}
 
 	for _, tt := range tests {
-		tt := tt
```

Comment:
Good suggestion. I just parallelized this test but the difference in time to run is non-noticeable. Since the added `tt := tt` and `t.Parallel()` don't appear to significantly increase test speed, will opt to not parallelize.

Without `t.Parallel()`
```
$ go test -run TestNewDataAvailabilityHeader -count=1
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	1.739s
$ go test -run TestNewDataAvailabilityHeader -count=2
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	3.031s
$ go test -run TestNewDataAvailabilityHeader -count=3
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	4.545s
``` 

With `t.Parallel()`
```
$ go test -run TestNewDataAvailabilityHeader -count=1
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	1.797s
$ go test -run TestNewDataAvailabilityHeader -count=2
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	3.057s
$ go test -run TestNewDataAvailabilityHeader -count=3
PASS
ok  	github.com/celestiaorg/celestia-app/pkg/da	4.412s
```

Reply:
Awesome :rocket: